requirements/base.txt

```diff
@@ -1,48 +1,54 @@
-# Wheel 0.25+ needed to install certain packages on CPython 3.5+
```

Comment:
This shouldn't be changed under any instance. You can add to it in the corresponding `local.txt` or `test.txt` This dependencies are fixed.